pkg/cluster/adminupdate_test.go

```diff
@@ -85,6 +85,20 @@ func TestAdminUpdateSteps(t *testing.T) {
 		"[Action renewMDSDCertificate]",
 	}
 
+	managedIdentityCertificateRenewalSteps := []string{
+		"[Action startVMs]",
+		"[Condition apiServersReady, timeout 30m0s]",
+		"[Action populateDatabaseIntIP]",
+		"[Action correctCertificateIssuer]",
+		"[Action fixMCSCert]",
+		"[Action fixMCSUserData]",
+		"[Action configureAPIServerCertificate]",
+		"[Action configureIngressCertificate]",
+		"[Action initializeOperatorDeployer]",
+		"[Action renewMDSDCertificate]",
+		"[Action ensureClusterMsiCertificate]",
+	}
```

Comment:
`managedIdentityCertificateRenewalSteps` is almost identical to `certificateRenewalSteps`, is there any way to deduplicate it?

```suggestion
	managedIdentityCertificateRenewalSteps := append(
	    certificateRenewalSteps,
		"[Action ensureClusterMsiCertificate]",
		)
```